projects/plugins/protect/src/js/components/credentials-needed-modal/index.jsx

```diff
@@ -26,15 +26,15 @@ const CredentialsNeededModal = () => {
 	useEffect( () => {
 		const interval = setInterval( () => {
 			if (
-				! credentialState.state ||
-				[ 'awaiting_credentials', 'unavailable' ].indexOf( credentialState.state ) >= 0
+				! Credential.state ||
```

Comment:
This logic needs to be updated to handle the newly expected value of `credential`.